test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -691,7 +691,7 @@ func runWorkload(b *testing.B, tc *testCase, w *workload) []DataItem {
 				b.Fatalf("op %d: %v", opIndex, err)
 			}
 			b.Cleanup(func() {
-				_ = nodePreparer.CleanupNodes(ctx)
```

Comment:
btw: should we mark the test as failed when CleanupNodes returns an error?
If some nodes aren't deleted due to some errors, the next test cases might be affected by those remaining Nodes.

Reply:
Fixed.